manifests/1.0.0/maistra.v1.0.0.clusterserviceversion.yaml

```diff
@@ -336,7 +336,7 @@ spec:
                   medium: Memory
               containers:
                 - name: istio-operator
-                  image: maistra/istio-operator-ubi8:0.12.0
+                  image: maistra/istio-operator-ubi8:1.0.0
```

Comment:
I think you need to rebase, as there are two of these now, ./manifests-maistra/ and ./manifests-servicemesh/.

Reply:
Ah, yes.. thanks... fixed it now.